common/src/com/intellij/plugins/haxe/util/HaxeCommonCompilerUtil.java

```diff
@@ -95,155 +100,229 @@ public static boolean compile(final CompilationContext context) {
       context.log(HaxeCommonBundle.message("module.0.is.excluded.from.compilation", context.getModuleName()));
       return true;
     }
+
+    if (!verifyProjectSettings(context)) {
+      return false;
+    }
+
+    final String workingPath = calculateWorkingPath(context);
+    if (! workingPath.isEmpty()) {
+      context.setErrorRoot(workingPath);
+    }
+    final File workingDirectory = new File(FileUtil.toSystemDependentName(workingPath));
+    if (!workingDirectory.exists()) {
+      if (!workingDirectory.mkdirs()) {
+        context.errorHandler(HaxeCommonBundle.message("output.path.not.found", workingPath));
+        return false;
+      }
+    }
+
+    final List<List<String>> commandLines = generateCommandLines(context);
+
+    final BooleanValueHolder hasErrors = new BooleanValueHolder(false);
+    try {
+      for (List<String> commandLine : commandLines) {
+
+        // XXX: Would it be useful to show the working directory in the output window??
```

Comment:
And output directory and full command line with arguments.

Reply:
I'm not sure that I have the output directory available.  That's up to haxe (or lime) and while I have some heuristics that seem to hold, I'm not sure what the compiler will actually do.  But I'll put out the extra info that I have.